package.json

```
@@ -33,6 +33,7 @@
     "lint": "eslint --ext .js,.ts,.tsx .",
     "lint:fix": "eslint --ext .js,.ts,.tsx --fix .",
     "type-check": "tsc",
+    "type-check:js": "tsc -p tsconfig.checkjs.json",
```

Always nice to be more explicit?

```suggestion
    "type-check:js": "tsc --project tsconfig.checkjs.json",
```